### PR TITLE
Fix leading slash exit in fuzz test

### PR DESCRIPTION
**Pull request by prestonvasquez**
## Summary

In CI we are getting the following:

```
[2024/09/13 15:55:05.224] tar: Removing leading `/' from member names
[2024/09/13 15:55:05.224] Command 'shell.exec' in function 'run-fuzz-tests' (step 2 of 2) failed: shell script encountered problem: exit code 1.
```

Suggest applying the solution here: https://unix.stackexchange.com/a/59244

## Background & Motivation

Fuzz tests are failing at tar.

### CODE REVIEW

File: .evergreen/run-fuzz.sh [hunk 1]
```diff
@@ -63,7 +63,7 @@ done
 # If the fuzz directory exists, then tar it up in preparation to upload to S3.
 if [ -d $PROJECT_DIRECTORY/fuzz ]; then
 	echo "Tarring up fuzz directory"
-	tar -czf $PROJECT_DIRECTORY/fuzz.tgz $PROJECT_DIRECTORY/fuzz
+	tar -czf $PROJECT_DIRECTORY/fuzz.tgz --absolute-names $PROJECT_DIRECTORY/fuzz
```
Comment (prestonvasquez) [inline .evergreen/run-fuzz.sh:66]: Note the fuzz test still fails in CI, but because of a legitimate issue.

Comment (blink1073) [reply .evergreen/run-fuzz.sh:66]: I think the proper way is to instead `cd $PROJECT_DIRECTORY` and then run `tar -cfz fuzz.tgz ./fuzz`

File: .evergreen/run-fuzz.sh [hunk 2]
```diff
@@ -63,7 +63,9 @@ done
 # If the fuzz directory exists, then tar it up in preparation to upload to S3.
 if [ -d $PROJECT_DIRECTORY/fuzz ]; then
 	echo "Tarring up fuzz directory"
-	tar -czf $PROJECT_DIRECTORY/fuzz.tgz $PROJECT_DIRECTORY/fuzz
+
+  	cd $PROJECT_DIRECTORY
+  	tar -cfz fuzz.tgz ./fuzz
```
Comment (blink1073) [inline .evergreen/run-fuzz.sh:68]:
```suggestion
  	tar cfz fuzz.tgz ./fuzz
```

**Review by blink1073 — APPROVED** [review]
LGTM!

**Pull request merged**